OPHD/States/MapViewStateUi.cpp

```diff
@@ -75,8 +75,6 @@ void MapViewState::initUi()
 	mFileIoDialog.hide();
 
 	mPopulationPanel.position({675, constants::ResourceIconSize + 4 + constants::MarginTight});
-	mPopulationPanel.population(&mPopulation);
-	mPopulationPanel.populationPool(&mPopulationPool);
```

Comment:
This could potentially have been grouped into the previous commit. Once initialization is done using constructor parameters, there is no longer a need to call the setter methods. Doing so initializes the values twice.